cypress/integration/IceBreaker.spec.js

```diff
@@ -0,0 +1,20 @@
+/// <reference types="cypress" />
+
+context('IceBreaker', () => {
+  before(() => {
+    cy.visit('/');
+    cy.get('[data-name=board-name-input]').type('IceBreaker Board Name');
+    cy.get('[data-name=more-settings-button]').click();
+    cy.get('[data-name=ice-breaker-question-input]').type(
+      'Testing ice breaker question'
+    );
+    cy.get('[data-name=create-button]').click();
+  });
+
+  it('should exist an ice breaker question', () => {
+    cy.get('[data-name=ice-breaker-message]').should(
+      'have.text',
+      'Testing ice breaker question'
+    );
+  });
+});
```

Comment:
one last thing, add this to the end of the tests so that it cleans up after itself (the tests run against the prod db).
```
  after(() => {
    cy.intercept('boards').as('getBoards');
    cy.visit('/');
    cy.wait('@getBoards');
    cy.get('[data-name=board-list-button]').click();
    cy.get('[data-name=delete-button]').click({ multiple: true });
    cy.get('[data-name=delete-confirm-button]').click({ multiple: true });
    cy.get('[data-name=board-table]').should('not.exist');
  });
```

Reply:
Nice. I'll be sending this in the next commit.